deploy/examples/create-external-cluster-resources.py

```diff
@@ -549,7 +549,7 @@ def _invalid_endpoint(self, endpoint_str):
                     f"IP address parts should be numbers: {ipv4}"
                 )
             intPart = int(eachPart)
-            if intPart < 0 or intPart > 254:
+            if intPart < 0 or intPart > 255:
```

Comment:
in the unit test you can check, can you add a small test
And do we need upstream fix for this in lower version

Reply:
I'm not planning to add a unit test for this as the future version already has an appropriate fix. And I don't think we are doing any release for the upstream version of this older version.